In [15]:
import pandas as pd

cases_data = pd.read_csv('covid_confirmed_usafacts.csv')
cases_df = pd.DataFrame(cases_data)

cases_df[cases_df.loc[:, 'State'] == 'GA']

countyFIPS            County Name State  StateFIPS  2020-01-22  \
397           0  Statewide Unallocated    GA         13           0   
398       13001        Appling County     GA         13           0   
399       13003       Atkinson County     GA         13           0   
400       13005          Bacon County     GA         13           0   
401       13007          Baker County     GA         13           0   
..          ...                    ...   ...        ...         ...   
552       13313      Whitfield County     GA         13           0   
553       13315         Wilcox County     GA         13           0   
554       13317         Wilkes County     GA         13           0   
555       13319      Wilkinson County     GA         13           0   
556       13321          Worth County     GA         13           0   

     2020-01-23  2020-01-24  2020-01-25  2020-01-26  2020-01-27  ...  \
397           0           0           0           0           0  ...   
398           0           0           0           0           0  ...   
399           0           0           0           0           0  ...   
400           0           0           0           0           0  ...   
401           0           0           0           0           0  ...   
..          ...         ...         ...         ...         ...  ...   
552           0           0           0           0           0  ...   
553           0           0           0           0           0  ...   
554           0           0           0           0           0  ...   
555           0           0           0           0           0  ...   
556           0           0           0           0           0  ...   

     2023-07-14  2023-07-15  2023-07-16  2023-07-17  2023-07-18  2023-07-19  \
397       86623       86623       86623       86623       86623       86623   
398        3846        3846        3846        3846        3846        3846   
399        1703        1703        1703        1703        1703        1703   
400        2853        2853        2853        2853        2853        2853   
401         343         343         343         343         343         343   
..          ...         ...         ...         ...         ...         ...   
552       30450       30450       30450       30450       30450       30450   
553         902         902         902         902         902         902   
554        1195        1195        1195        1195        1195        1195   
555        1887        1887        1887        1887        1887        1887   
556        2584        2584        2584        2584        2584        2584   

     2023-07-20  2023-07-21  2023-07-22  2023-07-23  
397       86623       86623       86623       86623  
398        3846        3846        3846        3846  
399        1703        1703        1703        1703  
400        2853        2853        2853        2853  
401         343         343         343         343  
..          ...         ...         ...         ...  
552       30450       30450       30450       30450  
553         902         902         902         902  
554        1195        1195        1195        1195  
555        1887        1887        1887        1887  
556        2584        2584        2584        2584  

[160 rows x 1269 columns]

Using Jupyter Notebook, I was able to use conditional selection to choose the state of Georgia. In the first row of data, the county shows as `Statewide Unallocated`. This means that the data is for the state as a whole. The final 7 days of data in that row show that the Covid-19 Case rate was stable at `86,623`.